dist/assests/css/footer.css

```diff
@@ -0,0 +1,51 @@
+.footer-section__subscrition .footer-title {
+  font-family: OpenSans;
```

Comment:
Why are you setting the font family again to open sans while it was already done in global.css , and actually it's a typo...it should be 'Open Sans' not OpenSans.
Same goes for lines 15 and 27

Reply:
Fixed